package.json

```diff
@@ -29,6 +29,6 @@
     }
   },
   "peerDependencies": {
-    "html-webpack-plugin": "2.x"
+    "html-webpack-plugin": "2.x || ^3"
```

Comment:
Expanding the peerDep range looks good to me (there's nothing in the [html-webpack-plugin changelog](https://github.com/jantimon/html-webpack-plugin/blob/master/CHANGELOG.md) that the template needs to be adjusted for).

Though perhaps for consistency the range should either use tilde for both v2 and v3, or else the `N.x` form for both?